Source/Logger/trace.cpp

```diff
@@ -15,12 +15,12 @@ namespace
 {
 
 //------------------------------------------------------------------------------
```

Comment:
This entire section can be deleted (I'm kind of surprised it even compiles right now)

Reply:
Removed

Reply:
Doesn't look you have fixed the places these functions were being called